<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>275</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="initialReportBtn">
       <property name="text">
        <string>Initial Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="requestRouteBtn">
       <property name="text">
        <string>Request Route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadRouteBtn">
       <property name="text">
        <string>Load Route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shareRouteBtn">
       <property name="text">
        <string>Share Route</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="objectionReportBtn">
       <property name="text">
        <string>Objection Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fmrBtn">
       <property name="text">
        <string>Fleet Management Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="licenseBtn">
       <property name="text">
        <string>License</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quartelyReportBtn">
       <property name="text">
        <string>Quartely Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>254</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openAction">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>initialReportBtn</tabstop>
  <tabstop>requestRouteBtn</tabstop>
  <tabstop>loadRouteBtn</tabstop>
  <tabstop>shareRouteBtn</tabstop>
  <tabstop>objectionReportBtn</tabstop>
  <tabstop>fmrBtn</tabstop>
  <tabstop>licenseBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
